Model/Film.php

```diff
@@ -5,12 +5,41 @@
 namespace Wtc\Film\Model;
 
 use Magento\Framework\Model\AbstractModel;
+use Wtc\Film\Api\Data\FilmInterface;
 
-class Film extends AbstractModel
+class Film extends AbstractModel implements FilmInterface
 {
     // This is enough to have a functional model
     public function _construct()
     {
         $this->_init(\Wtc\Film\Model\ResourceModel\Film::class);
     }
+
+    /**
+     * @return mixed|string|null
+     */
+    public function getTitle() {
```

Comment:
Use return type declaration instead of docblock and opening bracket should be on new line